app/views/_layouts/content-page.nunjucks

```diff
@@ -22,7 +22,7 @@
           {% endif %}
         </div>
 
-        <div class="column__one-third column__right">
+        <div class="column__one-third-right">
```

Comment:
I'm not 100% convinced on this but I see the argument for reducing the amount of media queries.